src/main/java/io/codelex/classesandobjects/practice/exercises/exercise7.java

```diff
@@ -0,0 +1,78 @@
+package io.codelex.classesandobjects.practice.exercises;
+
+
+class Dog{
+    private static String name;
+    private static String sex;
+    private static String motherName;
+    private static String fatherName;
+
+
+    public Dog(String name, String sex){
+        this.name = name;
+        this.sex = sex;
+        this.motherName = motherName;
+        this.fatherName = fatherName;
+    }
+
+    public static String getMotherName() {
+        return motherName;
+    }
+
+    public void setMotherName(String motherName) {
+        this.motherName = motherName;
+    }
+
+    public static String getFatherName() {
+        return fatherName;
+    }
+
+    public void setFatherName(String fatherName) {
+        this.fatherName = fatherName;
+    }
+
+    public static String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public static String getSex() {
+        return sex;
+    }
+
+    public void setSex(String sex) {
+        this.sex = sex;
+    }
+    static void displayDog(){
+        System.out.println(getName()+" "+getSex()+" "+getMotherName()+" "+getFatherName());
+    }
+}
+class dogTest{
+    String[][] dogs = {{"Max", "male"},{"Rocky", "male"},{"Sparky", "male"},{"Buster", "male"}, {"Sam", "male"},
```

Comment:
šos objektus vajadzetu izveidot atsevišķi
`Dog max = new Dog('max', 'male')` for example 
un tad pievienot array